Sources/SwiftFormat/Rules/OmitReturns.swift

```diff
@@ -0,0 +1,157 @@
+  }
+
+  public override func visit(_ node: ClosureExprSyntax) -> ExprSyntax {
+    let expr = super.visit(node)
+
+    // test { return ... }
+    if var closure = expr.as(ClosureExprSyntax.self),
+       let `return` = containsSingleReturn(closure.statements) {
+       closure.statements = unwrapReturnStmt(`return`)
+       diagnose(.omitReturnStatement, on: `return`, severity: .refactoring)
+       return ExprSyntax(closure)
+    }
+
+    return expr
+  }
+
+  private func transformAccessorBlock(_ accessorBlock: AccessorBlockSyntax) -> AccessorBlockSyntax? {
+    // We are assuming valid Swift code here where only
+    // one `get { ... }` is allowed.
+    switch accessorBlock.accessors {
+    case .accessors(let accessors):
+      guard var getter = accessors.filter({
+        $0.accessorSpecifier.tokenKind == .keyword(.get)
+      }).first else {
+        return nil
+      }
+
+      guard let body = getter.body,
+            let `return` = containsSingleReturn(body.statements) else {
+        return nil
+      }
+
+      guard let getterAt = accessors.firstIndex(where: {
+        $0.accessorSpecifier.tokenKind == .keyword(.get)
+      }) else {
+        return nil
+      }
+
+      getter.body?.statements = unwrapReturnStmt(`return`)
+
+      diagnose(.omitReturnStatement, on: `return`, severity: .refactoring)
+
+      return .init(
+          leadingTrivia: accessorBlock.leadingTrivia,
+          leftBrace: accessorBlock.leftBrace,
+          accessors: .accessors(accessors.with(\.[getterAt], getter)),
+          rightBrace: accessorBlock.rightBrace,
+          trailingTrivia: accessorBlock.trailingTrivia)
+
+    case .getter(let getter):
+      guard let `return` = containsSingleReturn(getter) else {
+        return nil
+      }
+
+      diagnose(.omitReturnStatement, on: `return`, severity: .refactoring)
+
+      return .init(
+          leadingTrivia: accessorBlock.leadingTrivia,
+          leftBrace: accessorBlock.leftBrace,
+          accessors: .getter(unwrapReturnStmt(`return`)),
+          rightBrace: accessorBlock.rightBrace,
+          trailingTrivia: accessorBlock.trailingTrivia)
+    }
+  }
+
+  private func containsSingleReturn(_ body: CodeBlockItemListSyntax) -> ReturnStmtSyntax? {
+    if let element = body.firstAndOnly?.as(CodeBlockItemSyntax.self),
+       let ret = element.item.as(ReturnStmtSyntax.self),
+       !ret.children(viewMode: .all).isEmpty, ret.expression != nil {
+      return ret
+    }
+
+    return nil
+  }
+
+  private func unwrapReturnStmt(_ `return`: ReturnStmtSyntax) -> CodeBlockItemListSyntax {
+    CodeBlockItemListSyntax([
+      CodeBlockItemSyntax(
+        leadingTrivia: `return`.leadingTrivia,
+        item: .expr(`return`.expression!),
+        semicolon: nil,
+        trailingTrivia: `return`.trailingTrivia)
+    ])
+  }
+}
+
+extension Finding.Message {
+  public static let omitReturnStatement: Finding.Message =
+    "`return` can be omitted because body consists of a single expression"
```

Comment:
```suggestion
    "remove 'return' because the body is a single expression"
```

Reply:
(As a style choice, we use single quotes instead of backticks, since that matched Swift compiler error messages at the time. However, it also looks like diagnostics have been introduced since then that use backticks...)